### PR TITLE
Product filters

### PR DESCRIPTION
**Pull request by RahullRawat**
Added filter functionality

**CSB Link** : https://b5xkj9.csb.app/product

### CODE REVIEW

File: src/pages/Products/utils/FilterReducer.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+				priceRange: action.payload,
+			};
+		}
+
+		// case "CATEGORY": {
+		// 	return {
+		// 		...state,
+		// 		category: action.payload,
+		// 	};
+		// }
```
Comment (AtulPant2704) [inline src/pages/Products/utils/FilterReducer.js]: You can remove these comments.

File: src/pages/Products/utils/FilterReducer.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+				sortBy: action.payload,
+			};
+
+		case "RESET":
+			return {
+				...state,
+				sortBy: null,
+				priceRange: null,
+				rating: null,
+				// category: null,
```
Comment (AtulPant2704) [inline src/pages/Products/utils/FilterReducer.js]: This as well.

**Review by AtulPant2704 — APPROVED** [review]
Very well formatted code with very good naming convention, made reviewing really really easy.

**Review by Abhijeet43 — APPROVED** [review]
Naming Conventions used are actually good!... Code is well formatted... UI is also good

**Pull request merged**